orca-keel/src/main/kotlin/com/netflix/spinnaker/orca/config/KeelConfiguration.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.config
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+import com.netflix.spinnaker.orca.KeelService
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.ComponentScan
+import org.springframework.context.annotation.Configuration
+import retrofit.Endpoint
+import retrofit.Endpoints
+import retrofit.RestAdapter
+import retrofit.client.Client
+import retrofit.converter.JacksonConverter
+
+@Configuration
+@ConditionalOnProperty("keel.enabled")
+@ComponentScan(
+  basePackages = [
+    "com.netflix.spinnaker.orca.keel",
+    "com.netflix.spinnaker.orca.keel.task",
+    "com.netflix.spinnaker.orca.keel.pipeline"
+  ]
+)
+open class KeelConfiguration {
+  @Bean open fun keelEndpoint(@Value("\${keel.baseUrl}") keelBaseUrl: String): Endpoint {
+    return Endpoints.newFixedEndpoint(keelBaseUrl)
+  }
+
+  @Bean open fun keelService(keelEndpoint: Endpoint,
+                  objectMapper: ObjectMapper,
```

Comment:
You'll want this to be `keelObjectMapper: ObjectMapper` so we wire in the correct one.